doc/source/ray-core/ray-generator.rst

```diff
@@ -162,12 +165,12 @@ Unblocking wait is possible with the Ray generator in the following ways:
 
 **Wait until a generator task completes**
 
-``StreamingObjectRefGenerator.completed()`` returns an object reference that is available when a generator task finishes or errors.
-For example, you can do ``ray.get(gen.compelted())`` to wait until a task completes. Note that using ``ray.get`` to ``StreamingObjectRefGenerator`` isn't allowed.
+``ObjectRefGenerator.completed()`` returns an object reference that is available when a generator task finishes or errors.
+For example, you can do ``ray.get(gen.compelted())`` to wait until a task completes. Note that using ``ray.get`` to ``ObjectRefGenerator`` isn't allowed.
```

Comment:
Just checking if there is a typo in the name of the method `ray.get(gen.completed())`

Reply:
fixed